### PR TITLE
Add `dkist` package

### PR DESCRIPTION
**Pull request by Cadair**
Checklist
- [x] Title of this PR is meaningful: e.g. "Adding my_nifty_package", not "updated meta.yaml".
- [x] License file is packaged (see [here](https://github.com/conda-forge/staged-recipes/blob/5eddbd7fc9d1502169089da06c3688d9759be978/recipes/example/meta.yaml#L64-L73) for an example).
- [x] Source is from official source.
- [x] Package does not vendor other packages. (If a package uses the source of another package, they should be separate packages or the licenses of all packages need to be packaged).
- [x] If static libraries are linked in, the license of the static library is packaged.
- [x] Package does not ship static libraries. If static libraries are needed, [follow CFEP-18](https://github.com/conda-forge/cfep/blob/main/cfep-18.md).
- [x] Build number is 0.
- [x] A tarball (`url`) rather than a repo (e.g. `git_url`) is used in your recipe (see [here](https://conda-forge.org/docs/maintainer/adding_pkgs.html#build-from-tarballs-not-repos) for more details).
- [x] GitHub users listed in the maintainer section have posted a comment confirming they are willing to be listed there.
- [x] When in trouble, please check our [knowledge base documentation](https://conda-forge.org/docs/maintainer/knowledge_base.html) before pinging a team.

### CODE REVIEW

File: recipes/dkist/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,64 @@
...
+about:
+  home: http://dkist.nso.edu
+  summary: DKIST User Tools
+  license: BSD-3-Clause
+  license_file: licenses/LICENSE.rst
+
+extra:
+  recipe-maintainers:
+    - Cadair
+    - SolarDrew
```
Comment (Cadair) [inline recipes/dkist/meta.yaml:64]: @SolarDrew please confirm you are ok with co-maintaining this :smile:

Comment (SolarDrew) [reply recipes/dkist/meta.yaml:64]: Yes, fine by me.

**Comment by Cadair** [comment]
@conda-forge/help-python I think this is ready for review. Thanks!

File: recipes/dkist/meta.yaml [hunk 2]
```diff
@@ -0,0 +1,64 @@
...
+    - dkist
+  commands:
+    - pip check
+    - pytest --pyargs dkist
+
+about:
+  home: http://dkist.nso.edu
+  summary: DKIST User Tools
+  license: BSD-3-Clause
+  license_file: licenses/LICENSE.rst
```
Comment (dopplershift) [inline recipes/dkist/meta.yaml:59]:
```suggestion
  license_file: licenses/LICENSE.rst
  dev_url: https://github.com/DKISTDC/dkist
```

File: recipes/dkist/meta.yaml [hunk 3]
```diff
@@ -0,0 +1,64 @@
...
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/dkist-{{ version }}.tar.gz
+  sha256: 54dc4a6bd60222b39f3ceb9a1e6a874b3ab2ec7466205ba48a98ff9a9ac81e04
+
+build:
+  skip: true  # [py<39]
```
Comment (dopplershift) [inline recipes/dkist/meta.yaml:13]: This looks like it's pure Python and could be noarch:
```suggestion
  noarch: python
```

File: recipes/dkist/meta.yaml [hunk 4]
```diff
@@ -0,0 +1,64 @@
...
+requirements:
+  host:
+    - python
+    - setuptools >=30.3.0
+    - setuptools-scm
+    - wheel
+    - pip
+  run:
+    - python
+    - appdirs >=1.4
```
Comment (dopplershift) [inline recipes/dkist/meta.yaml:26]: For noarch:
```suggestion
  host:
    - python >=3.9
    - setuptools >=30.3.0
    - setuptools-scm
    - wheel
    - pip
  run:
    - python >=3.9
    - appdirs >=1.4
```

**Review by dopplershift — CHANGES_REQUESTED** [review]
Just a few minor changes and this is good to go.

**Comment by Cadair** [comment]
@dopplershift ah yes, I forgot about noarch. Changes added, thanks!

**Review by dopplershift — APPROVED** [review]

**Pull request merged**